pyulog/ulog2csv.py

```diff
@@ -31,25 +32,29 @@ def main():
                         help='Output directory (default is same as input file)',
                         metavar='DIR')
     parser.add_argument('-i', '--ignore', dest='ignore', action='store_true',
-                        help='Ignore string parsing exceptions', default=False)
-
+                        help='Ignore string parsing exceptions', default=False)               
+    parser.add_argument(
+        '-t', '--time_s', dest='time_s', type = int, 
```

Comment:
Can you also add the time before parameter (`time_e`)?

Reply:
What do you mean exactly?

Reply:
You added a parameter to convert after a certain timestamp. The opposite makes sense as well: convert up to a certain timestamp. So I would add both options.